internal/mockexporter/loadtest.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"flag"
```

Comment:
I would use github.com/namsral/flag instead of standard library, 
it allows using config trough environment variable that is quite useful with containers

Reply:
Point taken, now config files can be read from env vars as well. In order to keep the `.go` file "dependency-less", I just made a simple wrapper for `flag` instead.